easybuild/easyconfigs/k/KMCLib/KMCLib-2.0-a2-foss-2023a-Python-2.7.18.eb

```diff
@@ -0,0 +1,51 @@
+easyblock = 'CMakeMake'
+
+name = 'KMCLib'
+version = '2.0-a2'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://github.com/leetmaa/KMCLib'
+description = """KMCLib - a general framework for lattice kinetic Monte Carlo (KMC) simulations"""
+
+toolchain = {'name': 'foss', 'version': '2023a'}
+
+source_urls = ['https://github.com/leetmaa/KMCLib/archive/']
+sources = ['v%(version)s.tar.gz']
+checksums = ['796620a67ad010df4b11734f703151c17441f82cef026f3d56386a34056d0213']
+
+builddependencies = [
+    ('binutils', '2.40'),
+    ('CMake', '3.26.3'),
+]
+
+dependencies = [
+    ('Python', '2.7.18'),
+    ('numpy', '1.16.6', versionsuffix),
```

Comment:
Using a "direct" numpy dependency here is different from what we usually do (using `SciPy-bundle`), but in this case it's a sensible way out, since the alternative would be creating a full `SciPy-bundle` with Python 2.7.18 (which requires a full `Python-bundle-PyPI` as well), which is overkill imho...

So, good as is for me.